### DISCUSSION TITLE
Release v0.4.1

### DISCUSSION THREAD

**Discussion by straussmaximilian** [MannLabs/alphapept#409] [Announcements]
Description will follow shortly

<hr /><em>This discussion was created from the release <a href='https://github.com/MannLabs/alphapept/releases/tag/v0.4.1'>Release v0.4.1</a>.</em>

**Comment by im281** [comment]
Hi @straussmaximilian  I see that there are a lot of open pull requests (from a year ago) Do you need help maintaining the code base?

**Reply by straussmaximilian** [reply]
Hi, yes there are several things that I could use help with. Feel free to get in touch.
Some of the PRs are intentionally on hold but I'll clean some of them up in the meantime.
